### PR TITLE
Responsive create wallet

### LINKED ISSUE #2051: Visual design update for the Seed Clipboard copy warning (Create new wallet screen)
Just a little reordering of paragraphs

**Old one:** 
image

**New one:**
image

Spec: https://xd.adobe.com/spec/fe3d450e-1d21-4667-6f73-1d5fbc65cf9d-880c/
Preview: https://xd.adobe.com/view/25a56ea5-239d-4b0d-5fdc-c6ee054c97b3-5394/

### PR DESCRIPTION
**Pull request by vctt94**
fixes #2051 and #2063 

There was a couple things I did not include.

- The button to show and hide the wallet information ("Wallets are deterministically generated...") when width is lower than 768px. Because the information is written in the markdown and I did not figure how to hide and show programatically;
- Move the input place of "I understand the risks" on the side on the markdown (same problem as before);
- Move the wallet name to the confirm seed part, because this will probably need a big refactor on how wallets are being created;

If having objections I can include them on this PR or we can have issues for that.

### CODE REVIEW

**Comment by matheusd** [comment]
I think it's fine leaving the expanded info for a second PR.

A couple of issues I'm seeing here:

- On the restore seed page, the "Private Passphrase" and "Confirm Private Passphrase" are unaligned.

- On the Create New Wallet page, the edges of the "Copy seed words to clipboard" are unaligned to the previous row of seed words.

- Related to previous issue: since you're touching this page, let's see if we can move away from using floats on the seed words section. Changing to using css grids is probably the best idea. Here's a sample config that should improve the css usage:

```css
.seedArea {
    /* margin & width */
    display: grid;
    grid-template-columns: repeat(5, 1fr); /* this will change to repeat(3, 1fr) on mobile */
}

.getstarted.content .seedWord {
  /* remove width & float, leave the rest as-is */
}

.getstarted.content .copy {
  /* remove width & float, leave the rest as-is and add: */
  grid-column: 4 / 6; /* this will change to 1 / 3 on mobile */
}
```

File: app/i18n/docs/en/Warnings/SeedCopy.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 
 It is also highly unadvised to maintain the seed on a computer file, especially without encryption, as that can lead to stealing of funds by anyone with access to the computer.
 
-Further, storing the seed on a computer file without backing it up on a physical medium (written down piece of paper stored in a secure location) can cause loss of funds if the local wallet file gets corrupted or is otherwise unavailable (due to hardware failure or any other number of issues).
+Further, storing the seed on a computer file without backing it up on a physical medium (written down piece of paper stored in a secure location) can cause loss of funds if the local wallet file gets corrupted or is otherwise unavailable (due to hardware failure or any other number of issues). If you want to learn more about seed security,
 
-If you want to learn more about seed security, please check our seed FAQ located at https://docs.decred.org/faq/wallets-and-seeds/.
+please check our seed FAQ located at https://docs.decred.org/faq/wallets-and-seeds/
```
Comment (matheusd) [inline app/i18n/docs/en/Warnings/SeedCopy.md]: Capitalize first word.

**Comment by matheusd** [comment]
Much better!
A final nit to fix on the modal: the top of the orange text ("Please note that copying...") is not aligned to the top of the second column when showing the two column mode.

You can add a rule to remove the `margin-top` of paragraphs inside the modal for the two to be aligned (the remaining `margin-bottom` is enough to add space between the paragraphs in the text).

**Comment by vctt94** [comment]
Removed the margin-top

**Pull request merged**